<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1038</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>711</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>481</width>
       <height>36</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>VM Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>99</width>
       <height>38</height>
      </rect>
     </property>
     <property name="text">
      <string>Help Find</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>30</y>
       <width>99</width>
       <height>38</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnGo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>541</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>GO : WARNING HAVE EVERYTHING SAVED THIS WILL LOG YOU OUT</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>711</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>481</width>
       <height>36</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>99</width>
       <height>38</height>
      </rect>
     </property>
     <property name="text">
      <string>Help Find</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>30</y>
       <width>99</width>
       <height>38</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>711</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>481</width>
       <height>36</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>VM X Config</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>99</width>
       <height>38</height>
      </rect>
     </property>
     <property name="text">
      <string>Help Find</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>30</y>
       <width>99</width>
       <height>38</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>711</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>481</width>
       <height>36</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Normal X Config</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>99</width>
       <height>38</height>
      </rect>
     </property>
     <property name="text">
      <string>Help Find</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>30</y>
       <width>99</width>
       <height>38</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>711</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>481</width>
       <height>36</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>EvDev Keyboard</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>99</width>
       <height>38</height>
      </rect>
     </property>
     <property name="text">
      <string>Help Find</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>30</y>
       <width>99</width>
       <height>38</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_6" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>711</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>481</width>
       <height>36</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Passthrough Mouse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>99</width>
       <height>38</height>
      </rect>
     </property>
     <property name="text">
      <string>Help Find</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>30</y>
       <width>99</width>
       <height>38</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_7" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>711</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>481</width>
       <height>36</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>VideoCard IOMMU Group</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>99</width>
       <height>38</height>
      </rect>
     </property>
     <property name="text">
      <string>Help Find</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>30</y>
       <width>99</width>
       <height>38</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Mouse Toggle</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionReload"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
